common/i18n.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * A string representation of each of the languages that we support and expect to have a translation
+ * file for.
+ */
+export enum TranslationLanguages {
```

Comment:
nit: not really the standard naming style for enums, this (and the one below) should be non-plural